file_sumHomemade.cpp

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+using namespace std;
+int main(){
+	int x, y;
+	cout << "Yo, proviamo a fare un piccolo sum\nInserisci due numeri\n>> ";
+	cin >> x;
+	cout << ">> ";
+	cin >> y;
+	cout << "Somma: " << x+y;
+	cout << "\nFigata eh?\n";
+}
```

Comment:
hai caricato un file.exe, cancellalo
inoltre, metti degli spazi, non scrivere tutto attaccato
sposta tutta l'implementazione della somma in una funzione a parte